main.cpp

```diff
@@ -186,6 +177,19 @@ struct Bass
 
     void playInstrument();
     void playOpenStrings(int numberOfStrings);
+    bool activeStatus() { return isActive; }
+    void reportActive()
+    {
+        std::cout << "This " + this->name + " is an ";
+        if (this->activeStatus())
+        {
+            std::cout << "active bass\n";
+        }
+        else
+        {
+            std::cout << "pass bass\n";
+        }
```

Comment:
If you were writing GUI code that needed to use correct english, you'd probably move that `an` into the `if/else` blocks, which could be simplified into a Ternary expression:
```
std::cout << "This " << this->name << " is ";
std::cout << (this->activeStatis() ? " an active bass\n" : "a passive bass\n");
```

Reply:
Awesome tip, thanks!